src/core/car.py

```diff
@@ -1,6 +1,16 @@
 from dataclasses import dataclass
 
-Point = tuple[int, int]
+@dataclass 
+class Point2D :
+
+    def __init__(self, x : float, y : float):
```

Comment:
I think the `dataclass` decorator automatically adds a init method, so that is not required.